modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/cards/Cards.tsx

```diff
@@ -229,12 +243,13 @@ interface ICardsSectionProps {
 function CardsSection({
 	cardsSection,
 	modalProps,
+	onClear,
 	onSelect,
 	saveButtonDisabled,
 }: ICardsSectionProps) {
 	const {field, label} = cardsSection;
 
-	const onClick = () => {
+	const onClickAssign = () => {
```

Comment:
I know it is a bit picky :smile:, but wouldn't be clear rename the handler using its purpose, like `handleAssign`, `assignField` as it is already the callback of an `onClick` prop?